es-vue-base/src/lib-mixins/forms.js

```diff
@@ -130,7 +139,13 @@ export default {
             this.submitInProgress = false;
         },
         getFields(obj, valueKey = 'obj') {
-            return Object.keys(obj)
+            let objKeys = null;
+            try {
+                objKeys = Object.keys(obj);
+            } catch (e) {
+                return [];
+            }
```

Comment:
`const objKeys = Object.keys(obj) || []`

Reply:
If `obj` is undefined, then a TypeError will get thrown. These error checks basically allow the mixin to be added to pages without throwing any errors regardless of configuration, whereas before omitting the validators section would cause errors.

Reply:
Here's the context: https://github.com/EnergySage/es-ds/pull/550/files#r1096872668